docs/projects/template-fps/setup-and-installing.md

```diff
@@ -1 +1,27 @@
-// TODO: Populate this page from content from: https://github.com/improbable/ECS-FPS/blob/master/docs/setup-and-installing.md
+# Set up the FPS Starter Project
+
+1. Follow the [machine setup guide]({{urlRoot}}/setup-and-installing#set-up-your-machine).
+2. Clone the `GDK-for-Unity-FPS-Starter-Project` repository:
+
+    |     |     |
+    | --- | --- |
+    | SSH | `git clone git@github.com:spatialos/GDK-for-Unity-FPS-Starter-Project.git` |
+    | HTTPS | `git clone https://github.com/spatialos/GDK-for-Unity-FPS-Starter-Project.git` |
+3. Setup dependencies by either:
```

Comment:
as this doc is in the unitygdk repo, should we specify that they need to go to the fps repo to run those scripts

Reply:
They won't be using this from the repo but will update to make it clear